packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -1,20 +1,22 @@
 {{#showBanner}}
```

Comment:
This is another nit that's fine to address later or maybe it won't even be a concern non-locally but I wonder if we can conditionally add this class on `body` based on if the feature is "on". Without the new env var update locally, I experience a very slight amount of jank sometimes and can see the body content moved down when the banner most definitely won't be rendered since the env var is 'none'.

<img width="1091" alt="image" src="https://github.com/mozilla/fxa/assets/13018240/16583af9-c382-4c72-aa71-fe27c6133db6">

Reply:
Ah good call. I just fixed this... I think when I moved this code to the afterRender block that got lost.